content/en/apps/tutorials/tasks.md

```diff
@@ -0,0 +1,122 @@
+    appliesTo: 'reports',
+    appliesToType: [‘assessment'],
+    appliesIf: function(contact, report) {
+      return r && r.fields.group_assessment && parseInt(r.fields.group_assessment)  > 3;
+    },
+    actions: [
+      {
+        type: 'report',
+        form: 'assessment_follow_up'
+      }
+    ],
+    events: [
+      {
+        id: 'cough-gt-3-days-follow-up',
+        days: 0,
+        start: 2,
+        end: 7
+      }
+    ],
+    resolvedIf: function(c, r, event, dueDate) {
+      return isFormArraySubmittedInWindow(c.reports, ['assessment_follow_up'], dueDate, event, null, r._id);
+    }
+}
+```
+
+### 2. Create an `assessment_follow_up` form as specified below referenced by the task you created. Refer to the app-forms tutorial.
+
+##### Data fields
+
+| type                          | name              | label                              | required | relevant            | appearance | constraint | constraint_message  | calculation | choice_filter  | hint | default |
+| ----------------------------- | ----------------- | ---------------------------------- | -------- | ------------------- | ---------- | ---------- | ------------------- | ----------- | -------------- | ---- | ------- |
+| begin group                   | group_assessment  | Assessment                         |          |                     |            |            |                     |             |                |      |         |
+| select_one yes_no             | visited_hf             | Did ${patient_name} go to a health facility for treatment? | yes      |                     |            |            |                     |             |                |      |         |
+| select_one progress_since_last_visit   | progress    | How is ${patient_name}'s condition since the last visit?     | yes      | ${visited_hf} = 'yes'    |            |            |                     |             |                |      |         |
+| end group                     |                   |                                    |          |                     |            |            |                     |             |                |      |         |
+
+##### Choices
+
+| list_name         | name | label           |
+| ----------------- | ---- | --------------- |
+| yes_no            | yes  | Yes             |
+| yes_no            | no   | No              |
+| progress_since_last_visit            | improved   | Improved              |
+| progress_since_last_visit            | same   | Stayed same              |
+| progress_since_last_visit            | worse   | Worsened              |
+
+##### Settings
+
+| form_title     | form_id    | version | style | path | instance_name  | default_language  |
+| -------------- | ---------- | ------- | ----- | ---- | -------------- | ----------------- |
+| Follow up| assessment_follow_up | 1       | pages | data |                | en                |
+
+##### Sub-folder
+
+```text
+forms/app
+```
+
+#### Build
```

Comment:
Perhaps we can break down this step into two separate ones. ie. 'Compile and Upload App Settings' and 'Convert and Upload Task Form'.

Reply:
Makes sense.